site/docs/v1/tech/identity-providers/samlv2-idp-initiated/index.adoc

```diff
@@ -0,0 +1,118 @@
+
+To create an Identity Provider navigate to [breadcrumb]#Settings -> Identity Providers# and click [breadcrumb]#Add provider# and select [breadcrumb]#SAML v2 IdP Initiated#.
+
+This will take you to the [breadcrumb]#Add SAML v2 IdP Initiated# panel. Here you will need to fill out the required fields. 
+
+image::identity-providers/identity-provider-samlv2-idp-initiated-add.png[Add SAML v2 IdP Initiated,width=1200,role=bottom-cropped]
+
+
+=== Form Fields
+
+[.api]
+[field]#Enabled# [Required]#Required#::
+Determines if this provider is enabled. If it is false then it will be disabled globally.
+
+[field]#Id# [optional]#Optional#::
+An optional UUID. When this value is omitted a unique Id will be generated automatically.
+
+[field]#Name# [required]#Required#::
+A unique name to identify the identity provider. This name is for display purposes only and it can be modified later if desired.
+
+[field]#Issuer# [required]#Required#::
+The EntityId (unique identifier) of the SAML v2 identity provider. This value should be provided to you.
+
+[field]#Use NameId for email# [optional]#Optional#::
+If this is enabled, FusionAuth will assume that the `NameID` in the SAML response contains the email address of the user.
+
+[field]#Email claim# [optional]#required# Required::
+The name of the email claim returned in the SAML response.
++
+When [field]#Use NameId for email# is enabled this field will not be displayed and will not be required.
+
+[field]#Verification key# [required]#Required#::
+The public key or certificate that you must import into FusionAuth's KeyMaster. This is the public key provided to you by the identity provider.
+
+=== Options
+
+image::identity-providers/identity-provider-samlv2-idp-initiated-add-options.png[Add SAML v2 Options section,width=1200,role=top-cropped]
+
+==== Form Fields
+
+[api]
+[field]#Reconcile lambda# [optional]#Optional#::
+A lambda maps custom claims returned from the SAML response into the FusionAuth `User` and `Registration`. To learn more about creating a lambda, view the link:/docs/v1/tech/lambdas/samlv2-response-reconcile/[SAML v2 Reconcile lambda documentation].
++
+To configure a lambda, navigate to [breadcrumb]#Settings -> Lambdas#.
+
+[field]#Debug# [optional]#Optional#::
+Some identity providers are not compliant with the SAML and XML signing specifications. This makes it challenging to get them working with FusionAuth. 
++
+If you are running into integration issues, toggle this setting on and FusionAuth will output debugging information into the Event Log during a SAML login. You can find the event log in [breadcrumb]#System -> Event Log#.
+
+== Integration Details
+
+After configuring the Identity Provider, you can view the details to see values that will likely be required by your SAML v2 Identity Provider in order to trust FusionAuth as a relying party. Do so by navigating to [breadcrumb]#Settings -> Identity Providers# and click the green magnifying glass on your SAML provider. 
```

Comment:
Consider 
After configuring the Identity Provider, FusionAuth will display values likely required by your SAML v2 Identity Provider to trust FusionAuth as a relying party. Do so by navigating to [breadcrumb]#Settings -> Identity Providers# and click the green magnifying glass on your SAML provider.